### ISSUE TITLE
Current DMG file for app 2.0 will not mount

### ISSUE THREAD

**Issue by paraviz02** [PeterKaminski09/baskup#18]
The file linked on the "app" website for version 2.0 will not mount - maybe it is corrupt?

**Comment by PeterKaminski09** [comment]
Could you provide a picture? I'm not able to replicate this problem, but I'll attach a link to another download if that helps.

**Comment by paraviz02** [comment]
I'm running macOS Sierra v10.12.3.  No idea why it would be spitting this error out.

![screen shot 2017-10-26 at 8 30 58 am](https://user-images.githubusercontent.com/6978717/32061979-05533dd0-ba28-11e7-849d-4bfdb9684bc1.png)

**Comment by PeterKaminski09** [comment]
Try this [link](https://www.dropbox.com/s/ac051umd8f1pj04/baskup-2.0-dmg.dmg?dl=0)

**Comment by paraviz02** [comment]
Hmm ... Maybe I am completely losing my mind.  That one came up with the same error.  The file size was about 1.4MB smaller than the original download.  Do you think the issue could be on my side?  Other DMG files open fine ...

**Comment by PeterKaminski09** [comment]
Hmm... maybe restart your computer? I have no idea what would cause that.

**Comment by couvares** [comment]
I'm having the same issue.

**Comment by petroseskinder** [comment]
I am also having the same issue. I downloaded disk image and it simply states no mountable file systems.

**Comment by PeterKaminski09** [comment]
I tried a different DMG creation software for this one. Maybe this will work?
 You'll need to drag the icon into your applications folder manually. 
https://www.dropbox.com/s/2muv9n5ajq50mwe/Baskup.app.dmg?dl=1

**Comment by PeterKaminski09** [comment]
If that doesn't work, try this video. It seems to be an issue with the OS version.

https://www.youtube.com/watch?v=gvOoOk2PS-Y

**Comment by petroseskinder** [comment]
The new dropbox linked sufficed for me. Thanks for the help.

**Comment by paraviz02** [comment]
Yep. That one works for me, too!  Thanks so much Peter, keep up the excellent work!

**Comment by couvares** [comment]
Works for me now too.  Thanks!

**Comment by PeterKaminski09** [comment]
I've updated the link on the [website](http://peterkaminski.me/baskup/). Could you let me know if that one opens for you all?

**Comment by petroseskinder** [comment]
dropbox links works, but new link does not.

**Comment by paraviz02** [comment]
New link on website works for me.

**Comment by petroseskinder** [comment]
New link on site works
